src/Visitors/VisitorCollector.php

```diff
@@ -49,80 +49,92 @@ public function getVisitor(ProblemsHolder $problemsHolder, AffectedInspections $
+                    fn (FieldDefinitionNode $fieldDefinitionNode) => $inspection->visitFieldDefinition(
+                        $problemsHolder,
+                        $fieldDefinitionNode
+                    ),
                     $inspection,
                     $affectedInspections,
                     $problemsHolder,
                 ),
                 NodeKind::INPUT_OBJECT_TYPE_DEFINITION => $this->wrapper(
-                    fn (InputObjectTypeDefinitionNode $node) =>
-                        $inspection->visitInputObjectTypeDefinition($problemsHolder, $node),
+                    fn (InputObjectTypeDefinitionNode $node) => $inspection->visitInputObjectTypeDefinition(
+                        $problemsHolder,
+                        $node
+                    ),
                     $inspection,
                     $affectedInspections,
                     $problemsHolder,
                 ),
                 NodeKind::ARGUMENT => $this->wrapper(
-                    fn (ArgumentNode $argumentNode) =>
-                        $inspection->visitArgumentNode($problemsHolder, $argumentNode),
+                    fn (ArgumentNode $argumentNode) => $inspection->visitArgumentNode($problemsHolder, $argumentNode),
                     $inspection,
                     $affectedInspections,
                     $problemsHolder,
                 ),
                 NodeKind::OBJECT_TYPE_DEFINITION => $this->wrapper(
-                    fn (ObjectTypeDefinitionNode $node) =>
-                        $inspection->visitObjectTypeDefinition($problemsHolder, $node),
+                    fn (ObjectTypeDefinitionNode $node) => $inspection->visitObjectTypeDefinition(
+                        $problemsHolder,
+                        $node
+                    ),
                     $inspection,
                     $affectedInspections,
                     $problemsHolder,
                 ),
                 NodeKind::LIST_TYPE => $this->wrapper(
-                    fn (ListTypeNode $node, Node $parent, array $ancestors) =>
-                        $inspection->visitListType($problemsHolder, $node, $parent, $ancestors),
+                    fn (ListTypeNode $node, Node $parent, array $ancestors) => $inspection->visitListType(
+                        $problemsHolder,
+                        $node,
+                        $parent,
+                        $ancestors
+                    ),
                     $inspection,
                     $affectedInspections,
                     $problemsHolder,
                 ),
                 NodeKind::ENUM_TYPE_DEFINITION => $this->wrapper(
-                    fn (EnumTypeDefinitionNode $node) =>
-                    $inspection->visitEnumTypeDefinition($problemsHolder, $node),
+                    fn (EnumTypeDefinitionNode $node) => $inspection->visitEnumTypeDefinition($problemsHolder, $node),
```

Comment:
I can revert the reformatting if you want, but I found the existing formatting somewhat misleading here.